app/components/dex-create.jsx

```diff
@@ -71,13 +80,28 @@ export class DexCreate extends Component {
             </div>
             <div className="form-group">
               <label htmlFor="generation">Generation</label>
-              <select className="form-control" ref={(c) => this._generation = c} defaultValue="7">
+              <select className="form-control" onChange={this.handleChange} value={this.state.generation}>
                 <option value="7">Seven</option>
                 <option value="6">Six</option>
               </select>
               <i className="fa fa-chevron-down" />
             </div>
             <div className="form-group">
+              <label htmlFor="region">Regionality</label>
+              <div className="radio">
+                <label>
+                  <input type="radio" name="region" checked={region === 'national'} value="national" onChange={() => this.setState({ ...this.state, region: 'national' })} />
```

Comment:
i know we have it in several other places (cause i thought it was necessary), but you're `handleChange` function showed me that `...this.state` isn't necessary.

i thought `this.setState` overwrote the value of `this.state` so whenever we called it, we had to make sure we included all the other props (like `loading` and `error` and stuff) and thats what `...this.state` is doing. but `this.setState` actually takes what you pass into it and merges it with the current state so we dont have to do it.